jOOQ/src/main/java/org/jooq/InsertOnConflictDoActionStep.java

```diff
@@ -59,11 +59,17 @@
  *
  * @author Lukas Eder
  */
-public interface InsertOnConflictDoUpdateStep<R extends Record> {
+public interface InsertOnConflictDoActionStep<R extends Record> {
 
     /**
      * Add the <code>DO UPDATE</code> clause.
      */
     @Support({ POSTGRES_9_5 })
     InsertOnDuplicateSetStep<R> doUpdate();
+
+    /**
+     * Add the <code>DO IGNORE</code> clause.
+     */
+    @Support({ POSTGRES_9_5 })
+    InsertFinalStep<R> doIgnore();
 }
```

Comment:
The actual syntax in PostgreSQL is actually `DO NOTHING`. I would prefer if we repeat that in our API as well